packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -47,6 +47,15 @@
                     </f-button>
                 </slot>
 
+                <component
+                    :is="titleHtmlTag"
+                    v-if="title"
+                    :class="$style['c-megaModal-title']"
```

Comment:
Could you also add the "plain" CSS class here so that applications can target it? e.g. `['c-megaModal-title', $style['c-megaModal-title']`.